frontend/src/apis/index.js

```diff
@@ -18,7 +28,9 @@ const myFetch = async (url, config = {}) => {
         } catch (error) {
             console.log('Not a json :)');
         }
-    } else throw new Error(response.status);
+    } else {
+        throw new FetchException(response.code, await response.json());
```

Comment:
why do you need await here? we already waiting for the response on line 16.

Reply:
It's from the fact that `json()` returns a `Promise` as defined here [https://developer.mozilla.org/en-US/docs/Web/API/Body/json](url)

Also the previous line has `await` before `response.json()` 
```
  try {
      return await response.json();
      } catch (error) {
           console.log('Not a json :)');
      }
```